src/Microsoft.Extensions.Configuration.Binder/project.json

```diff
@@ -6,7 +6,8 @@
     "nowarn": [ "CS1591" ],
     "xmlDoc": true
   },
-  "description": "Functionality to bind an object to data in configuration providers.",
+  "description": "Functionality to bind an object to data in configuration providers using the Microsoft.Extensions.Configuration.IConfiguration interface.",
```

Comment:
I like mentioning the interface because it provides a bit more context to these packages. I didn't want it to be ultra generic, e.g. "Binds configuration data to objects." That statement would be true, but doesn't tie it to this whole new config system. Thoughts?